modules/rn_3_10_0.adoc

```diff
@@ -15,6 +15,11 @@ For more information, see the link:https://access.redhat.com/support/policy/upda
 [id="new-features-and-enhancements-310"]
 == {productname} new features and enhancements
 
+[id="ibm-power-z-linuxone-support"]
+==== IBM Power, IBM Z, IBM® LinuxONE support:
+
+With this release, IBM Power (ppc64le), IBM Z (390x), and IBM® LinuxONE (s390x) architectures are supported.
```

Comment:
IBMZ (390x) is wrong. it will be IBM Z(s390x)

Reply:
Ack